src/Catalyst.Core.Modules.Web3/ApiModule.cs

```diff
@@ -179,5 +160,30 @@ public void Configure(IApplicationBuilder app)
                 app.UseSwaggerUI(swagger => { swagger.SwaggerEndpoint("/swagger/v1/swagger.json", "Catalyst API"); });
             }
         }
+
+        private sealed class SharedContainerProviderFactory : IServiceProviderFactory<ContainerBuilder>
+        {
+            readonly IContainer _container;
+
+            public SharedContainerProviderFactory(IContainer container)
+            {
+                _container = container;
+            }
+
+            public ContainerBuilder CreateBuilder(IServiceCollection services)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'CreateBuilder'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)